src/EditorFeatures/Test/CodeLens/AbstractCodeLensTest.cs

```diff
@@ -16,6 +16,9 @@ public abstract class AbstractCodeLensTest
         {
             using (var workspace = await TestWorkspace.CreateAsync(input))
             {
+                var service = workspace.GetService<ICodeLensReferencesService>();
```

Comment:
the old way seems right way to test. if you really want to use GetService, you need to put export of ICodeLensRferencesService in TestExports